File: [Lv.0]/커피 심부름/커피 심부름_sooyeon.js
function solution(order) {
    let latte = 0;
    order.map((v)=>{
      if(v.includes("latte")){
        latte ++;
      }
    })
    console.log(latte * 5000 + (order.length - latte) * 4500);
    return (latte * 5000 + (order.length - latte) * 4500);
}

solution(["cafelatte", "americanoice", "hotcafelatte", "anything"])

File: [Lv.0]/커피 심부름/커피 심부름_sooyeon.cs
using System;

public class Solution
{
  public int solution(string[] order)
  {
    int num = 0;
    foreach (var item in order)
    {
      if (item.Contains("latte"))
      {
        num++;
      }
    }
    return (num * 5000 + (order.Length - num) * 4500);
  }
}